databags/databag_timer.cpp

```diff
@@ -0,0 +1,55 @@
+#include "databag_timer.h"
+
+void TimersDatabag::_bind_methods() {
+}
+
+void TimersDatabag::set_now(const uint64_t newNow){
+	now = newNow;
+}
+
+void TimersDatabag::internal_restart_precise_timer(const godex::TimerHandle timer, const uint64_t microseconds) {
+	timers[timer] = microseconds;
+}
+
+godex::TimerHandle TimersDatabag::new_timer(const real_t seconds) {
+	return new_precise_timer((uint64_t)((double)seconds * 1000000.0));
```

Comment:
Why cast to `double`? However, in general is safer to use `static_cast` instead to rely on C cast, so the compiler is able to report eventual warnings or errors.
 
Here we could use the syntax `1'000'000.0` so it's more readable.